### DISCUSSION TITLE
Help with setting up scheduled task

### DISCUSSION THREAD

**Discussion by ray-bun** [coollabsio/coolify#2477] [General]
I have below set up in vercel, how do I do the same with coolify

`{
  "crons": [
    {
      "path": "/download/xxx",
      "schedule": "0 */6 * * *"
    },
    {
      "path": "/download/bbb",
      "schedule": "0 */6 * * *"
    }
  ]
}`

**Comment by ray-bun** [comment]
Sample command that is working for me
``
curl --max-time 600 -o /dev/null https://qocw8ww.xxx.com/download/xxx
`

**Comment by backurdi** [comment]
Hey @ray-bun,  I am stuck with migrating my Vercel cron-jobs. Using Curl does not work for me, I get failed on the event? Did you add anything else? And what kind of project is it?

Any help would be appreciated.
